uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/directives/leave-request-popup_test.js

```diff
@@ -0,0 +1,56 @@
+(function (CRM) {
+  define([
+    'common/angular',
+    'common/angularMocks',
+    'leave-absences/shared/config',
+    'leave-absences/shared/directives/leave-request-popup',
+  ], function (angular) {
+    'use strict';
+
+    describe('leaveRequestPopup', function () {
+      var $compile, $log, $rootScope, directive, $uibModal,
+        $controllerScope, $provide, DateFormat;
+
+      /**
+       * Creates and compiles the directive
+       */
+      function compileDirective() {
```

Comment:
Put this at the bottom, leverage hoisting to get simple utility functions out of the way